### DISCUSSION TITLE
Query from API

### DISCUSSION THREAD

**Discussion by runa-danishdin** [lightdash/lightdash#6181] [Q&A] [answered]
Hi Team,

I am confused by the documentation regarding running a query from API
https://docs.lightdash.com/api/v1/#tag/Exploring/operation/postRunQuery

Say I have a Lightdash and there is a table called products.
From the UI, I run the SQL command
"Select codes from products" 
Which returns 1 column with all of the product codes.

How do I run this from the API?

**Comment by owlas** [comment, marked as answer]
Hey @runa-danishdin you can run a raw sql query by posting to this endpoint, this was missing from the documentation, which I've fixed on this PR: https://github.com/lightdash/lightdash/pull/6296

```
POST /api/v1/projects/{projectUuid}/sqlQuery

{
  "sql": "select codes from products"
}
```

**Reply by owlas** [reply]
Hey @runa-danishdin this is fixed now btw: https://github.com/lightdash/lightdash/pull/6296
